README.md

```diff
@@ -1448,10 +1455,10 @@ satisfy the consolidated requirements of:
 ```
 
 
-RRM Rights are just links between `Manifestation`s and licenses. Licenses are usually intended to be
-read by humans, so the license should be stored on a technology that prevents changes to the
-license. This could be an immutable ledger or blockchain, or be linked by Content-Addressing. With
-this in mind, an implementation in IPLD/IPFS is favored:
+The `Right` can be seen as the link between a `Manifestation` and its licenses. To prevent
+undetected changes to these linked licenses, the licenses would ideally be stored on an immutable
+ledger or content-addressed storage layer. With this in mind, the implementation in IPLD (on IPFS)
+is favoured:
```

Comment:
We removed "Licenses are usually intended to be read by humans".
I think this statement should still be included somehow.